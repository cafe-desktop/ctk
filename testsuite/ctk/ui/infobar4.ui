<!-- invalid element -->
<interface>
  <object class="CtkInfoBar">
    <child>
      <object class="CtkButton" id="button_cancel"/>
    </child>
    <action-widgets>
      <worbl/>
    </action-widgets>
  </object>
</interface>
