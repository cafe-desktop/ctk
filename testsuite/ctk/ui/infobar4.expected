ERROR: ctk-builder-error-quark 1
.:8:15 Unsupported tag for CtkInfoBar: <worbl>
